.github/workflows/chart-release.yaml

```diff
@@ -43,6 +43,10 @@ on:
         type: string
         description: "image tag (required if image_tag_generation: specified)"
         required: false
+      helm_repo_list:
+        type: string
+        description: "List of helm repos to add, in format: repo_name1,repo_url1,repo_name2,repo_url2"
+        required: false
```

Comment:
i think a default value of '' is missing here, otherwise a error should get thrown, if no helm_rpo_list is provided